Gemfile

```diff
@@ -25,7 +25,8 @@ end
 
 gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 1.6', '< 3'])
-gem "hiera", *location_for(ENV['HIERA_LOCATION'] || '~> 1.0')
+# gem "hiera", *location_for(ENV['HIERA_LOCATION'] || '~> 1.0')
+gem "hiera", :path => "#{File.dirname(File.dirname(__FILE__))}/hiera", :require => false
```

Comment:
yeah ! Actually - we will not need these changes as we are instead going to delete the puppet_backend - it is a horrible thing anyway and creates a circular dpendency between puppet and hiera repos.